new-lamassu-admin/src/pages/Blacklist/Blacklist.js

```diff
@@ -172,6 +232,33 @@ const Blacklist = () => {
                 ? `${clickedItem.display} blacklisted addresses`
                 : ''}{' '}
             </H4>
+            <Box
+              display="flex"
+              alignItems="center"
+              justifyContent="end"
+              mr="-140px">
+              <P>Enable paper wallet (only)</P>
+              <Switch
+                checked={enablePaperWalletOnly}
+                onChange={e =>
+                  enablePaperWalletOnly
+                    ? addressReuseSave({
+                        enablePaperWalletOnly: e.target.checked
+                      })
+                    : setConfirmDialog(true)
+                }
+                value={enablePaperWalletOnly}
+              />
+              <Label2>{enablePaperWalletOnly ? 'On' : 'Off'}</Label2>
+              <Tooltip width={304}>
+                <P>
+                  The "Enable paper wallet (only)" option means that all only
+                  paper wallets paper wallets will be printed for users, and
```

Comment:
`paper wallets paper wallets` on repeat here. And `... means that all only ...` doesn't sound well